youtube_dl/extractor/kaltura.py

```diff
@@ -373,5 +373,5 @@ def sign_url(unsigned_url):
             'duration': info.get('duration'),
             'timestamp': info.get('createdAt'),
             'uploader_id': info.get('userId') if info.get('userId') != 'None' else None,
-            'view_count': info.get('plays'),
+            'view_count': int(info.get('plays')) if info.get('plays') is not None else None,
```

Comment:
This is needed because the play count was received as an integer string, e.g. `"47"` rather than `47` for some reason.

Reply:
Note that I've now changed this line of code to use `int_or_none`.